### DISCUSSION TITLE
Inaudible sound on one of my BirdNet-Pi set ups

### DISCUSSION THREAD

**Discussion by tomwhite101** [mcguirepr89/BirdNET-Pi#408] [Q&A]
Hi all,
I wonder if I can ask the group for some advise.
My first BirdNet Pi has been running now for two months with relatively little issue. It runs on my office network and I am able to use all the functions and listen back to recordings happily with that set up. It is on a RPi 3b+ with this microphone doing a great job: https://www.amazon.co.uk/dp/B09N6YR9LM/ref=cm_sw_r_awdo_3Y1Y10ZGQGQF3NZQE98R_0

At home I have another installation. 
This has been running for about a month doing a great job on identifications with thousands of records at 0.7certainty.
However - the sound files are inaudibly quiet - I can see the calls on the spectrograms but when I replay the calls I cannot hear most of them - if there is a very loud sharp call, like a crow, I can just make it out.
This installation is on a RPi4 and since I set it up it has been using a this mic https://thepihut.com/products/mini-usb-microphone on a 3m USB extension. So I assumed this cheap crappy mic was the source of the problem.

With this in mind my Dad very kindly soldered me up a very neat EM272 set up which I have twinned with a UGREEN USB sound card (https://www.amazon.co.uk/UGREEN-Headphone-Microphone-Raspberry-Ultrabook/dp/B01N905VOY/ref=sr_1_3?keywords=ugreen+sound+card&qid=1656328814&sprefix=ugreen+so%2Caps%2C256&sr=8-3)

With this new mic set up detections have increased, but I am still unable to really hear the calls on playback - I have tried different devices which I know work with the setup I have on my other installation (iPhone and Laptops) but to no avail.

So... What am I missing - is there some hidden setting I need to fiddle with?

Many thanks.

**Comment by Svardsten53** [comment]
I have two PUI 5024 microphones which via a Scarlett 2i2 are connected to an RPi 4+. Sometimes I also get detections that are not heard at all and then it is often seen very faintly in the spectrogram. It is probably the case that the analysis in BirdNET-Pi "hears" better than what the human ear does which means we sometimes can't hear detections.

You can also increase the level of the microphone input with the alsamixer command in RPi, which can make the sounds easier to hear.

**Reply by tomwhite101** [reply]
This is consistent all the time. I will look in to that. Many thanks.

**Comment by ehpersonal38** [comment]
Maybe try this and see if it helps? https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card

**Reply by tomwhite101** [reply]
@ehpersonal38 This worked to some extent. I now have audible recordings!
They are not as clear as my other set up but pretty good.
Interestingly I now have very define horizontal stripes on my spectrograms exactly aligned with the kHz increments. Do you think this is a sign of interference, it seems I have dialled it up what ever the cause.

**Comment by nilspupils** [comment]
Hi @tomwhite101: 
I have a suspicion what could be the cause of your low microphone output, which is a low bias voltage at the microphone. Could you do a voltage measurement across the capsule (while it is on)? I have a similar problem with one of my test setups using the exact same soundcard as you have. I think it doesn't work with the common capsules like EM272 and AOM-5024 so well because they need a bigger bias voltage than this soundcard provides. They "work" but you loose a lot of sensitivity, which is the characteristic the capsule was chosen for in the first place. If you cold provide some more real world data (voltage measurements) we can hopefully get to the source of the problem. 
Thanks, 
Nils

**Reply by tomwhite101** [reply]
I have a multi meter but no idea really how to wrangle it. could you easily describe what/where I would measure.

The easies test might be to swap out the UGREEN part.
Thanks, 
Tom

**Reply by nilspupils** [reply]
Put your multimeter in voltage mode on 2V (or 20 V) range. Then connect each probe to one contact of the capsule. You should see a reading of 0.4-3V.

**Reply by tomwhite101** [reply]
@nilspupils i couldn't actually get at it but my dad says he checked it before completing and it was giving 2.8 so it doesn't seem that should be an issue?

**Reply by nilspupils** [reply]
If that turns out to be the case it is an excellent value (the best i have measured so far) and should be well within the specs of the capsule. This can then be ruled out as the reason for your low output. 
I am a little puzzled because i had much lower readings with the exact same interface, but it could be they have changed some components. 
Last thing that comes to my mind is a temperature damage of the capsule from soldering, which can reduce the sensitivity. 
Best, 
Nils

**Comment by mcguirepr89** [comment]
Hi, @tomwhite101 I only haven't weighed in because you're in good hands with these folks ~~ :+1:
